public/demo-survey/config.json

```
@@ -16,16 +16,16 @@
     },
     "uiConfig": {
         "contactEmail": "test@test.com",
-        "helpTextPath": "survey-demo/help.md",
-        "logoPath": "assets/revisitLogoSquare.svg",
+        "helpTextPath": "demo-survey/assets/help.md",
+        "logoPath": "demo-survey/assets/revisitLogoSquare.svg",
```

Just marking this once instead of in all configs, but we should change this to

```suggestion
        "logoPath": "revisitAssets/revisitLogoSquare.svg",
```